homeassistant/components/bthome/sensor.py

```diff
@@ -260,6 +263,10 @@
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    # Text (-)
+    (BTHomeExtendedSensorDeviceClass.TEXT, None): SensorEntityDescription(
+        key=f"{BTHomeExtendedSensorDeviceClass.TEXT}",
```

Comment:
This could probably be `str()` wrapped instead since nothing is being formatted, but it probably doesn't make much difference

Reply:
Fixed (for all sensors without unit)